.github/labeler.yml

```diff
@@ -0,0 +1,185 @@
+# Package labeling rules
+
+"[Package] A11y":
+- packages/a11y/**
+#"[Package] ":
```

Comment:
Some packages don't have an associated label. This is direct to solve, and labels should be created later for these packages, but I didn't want to interfere with the WIP audit in https://github.com/WordPress/gutenberg/issues/52583.